optuna/samplers/_nsga2/__init__.py

```diff
@@ -0,0 +1 @@
+from optuna.samplers._nsga2.sampler import NSGAIISampler  # NOQA
```

Comment:
How about keeping it blank as previously mentioned to keep indirections minimal.
```suggestion
```

Reply:
If it is kept blank, won't samplers that used to be importable with `from optuna.samplers._nsga2 import NSGAIISampler` become unimportable?

Reply:
Sorry, what samplers are you referring to? `_nsga2/` has always been private so it shouldn't be an issue. We'd have to update a single line in addition in `samplers/__init__.py`.